### ISSUE TITLE
Qwen2.5-7B-Instruct 微调 loss 和 梯度问题

### ISSUE THREAD

**Issue by ZY-LUCK** [datawhalechina/self-llm#362]
### 训练过程中，loss 为 0，梯度为 nan
{'loss': 418.7236, 'grad_norm': nan, 'learning_rate': 9.856938483547926e-05, 'epoch': 0.04}
{'loss': 0.0, 'grad_norm': nan, 'learning_rate': 9.713876967095852e-05, 'epoch': 0.09}
...

**Comment by KMnO4-zx** [comment]
可能是peft或transformers包版本的问题，可以尝试使用教程中提供的镜像再次进行训练~
